setup/src/Magento/Setup/Validator/AdminCredentialsValidator.php

```diff
@@ -29,21 +30,29 @@ class AdminCredentialsValidator
      */
     private $setupFactory;
 
+    /**
+     * @var DriverOptions
+     */
+    private $driverOptions;
+
     /**
      * Initialize dependencies.
      *
      * @param \Magento\Setup\Model\AdminAccountFactory $adminAccountFactory
      * @param \Magento\Setup\Module\ConnectionFactory $connectionFactory
      * @param \Magento\Setup\Module\SetupFactory $setupFactory
+     * @param DriverOptions|null $driverOptions
      */
     public function __construct(
         \Magento\Setup\Model\AdminAccountFactory $adminAccountFactory,
         \Magento\Setup\Module\ConnectionFactory $connectionFactory,
-        \Magento\Setup\Module\SetupFactory $setupFactory
+        \Magento\Setup\Module\SetupFactory $setupFactory,
+        DriverOptions $driverOptions
```

Comment:
Please add a new dependency in a backward compatible way: last optional parameter, the property should be initialized using object manager if it was not passed to the constructor

Reply:
It also should have default value `null` make it optional.

Reply:
I have just sent a commit to make it optional.